packages/dashboard/src/portal/components/VotePopup.vue

```diff
@@ -44,6 +55,13 @@ export default class VotePopup extends Vue {
     this.openProposalDialog = true;
   }
 
+  async getUserNodes(farmIDs: any[]): Promise<any> {
+    const url = `${config.gridproxyUrl}/nodes?ret_count=true&page=1&size=10&farm_ids=${farmIDs}`;
```

Comment:
we need also to make sure these nodes are online
```suggestion
    const url = `${config.gridproxyUrl}/nodes?status=up&ret_count=true&page=1&size=10&farm_ids=${farmIDs}`;
```